### ISSUE TITLE
Chaos executor

### ISSUE THREAD

**Issue by jasmingacic** [kubeshop/testkube#462]
Add ability to testkube to spin up additional pod that would run in parallel with execution.

Benefits for QA would be that on top of functionality testing, testkube could help identify potential infrastructure flaws. 

for example test target is a REST API deployed in the same cluster testkube would run standard script execution and in parallel would try and kill the same pod. We would confirm how does the REST API is behaving in case of any outages e.g. scaling, recover, down time...

**Comment by dimonlex** [comment]
I'd like to be part of the discussion on this before the work gets started (if/when)

**Comment by jasmingacic** [comment]
Let's schedule something!!!

**Comment by TheBrunoLopes** [comment]
won't do.
